src/Razor/src/Microsoft.AspNetCore.Razor.OmniSharpPlugin/MSBuildProjectManager.cs

```diff
@@ -193,7 +194,13 @@ private void UpdateProjectState(ProjectInstance projectInstance)
         foreach (var documentFilePath in projectSnapshot.DocumentFilePaths)
         {
             OmniSharpHostDocument? associatedHostDocument = null;
-            var currentHostDocument = projectSnapshot.GetDocument(documentFilePath).HostDocument;
+            var document = projectSnapshot.GetDocument(documentFilePath);
```

Comment:
Is this because of nullablilty changes? Can we do them in a separate PR, for clarity?